src/movie_lib.cc

```diff
@@ -2794,4 +2792,19 @@ static void _nfPkDecomp(unsigned char* a1, unsigned char* a2, int a3, int a4, in
     }
 }
 
+constexpr uint16_t loadUInt16LE(const uint8_t* b)
+{
+    return (b[1] << 8) | b[0];
+}
+
+constexpr uint32_t loadUInt32LE(const uint8_t* b)
+{
+    return (b[3] << 24) | (b[2] << 16) | (b[1] << 8) | b[0];
+}
+
+uint8_t getOffset(uint16_t v)
```

Comment:
@alexbatalov As I mentioned in https://github.com/alexbatalov/fallout1-ce/pull/117#issuecomment-1986828872, the return type here (changed from `auto` to `uint8_t`) is incorrect, it should be `int` or `int32_t`